### PR TITLE
added simple parallax to item details

### PR DESCRIPTION
**Pull request by kleinsenberg**
Simple parallax effect for item details view.

Will iterate on this and add description.

### CODE REVIEW

File: app/src/main/java/com/nabilhachicha/kc/ItemDetailActivity.java [hunk 1]
```diff
@@ -74,14 +73,20 @@ protected void onCreate(Bundle savedInstanceState) {
         // populate fields
         ImageView imageView = (ImageView) findViewById(R.id.itemImg);
         mPicasso.load(mVenue.getImageUrl()).resize(screenWidth, screenWidth).centerInside().into(imageView);
-        ((TextView) findViewById(R.id.textName)).setText(mVenue.getName());
-        ((TextView) findViewById(R.id.textDescription)).setText(mVenue.getDescription());
+//        ((TextView) findViewById(R.id.textName)).setText(mVenue.getName());
+//        ((TextView) findViewById(R.id.textDescription)).setText(mVenue.getDescription());
```
Comment (JamesScottTAB) [inline app/src/main/java/com/nabilhachicha/kc/ItemDetailActivity.java:77]: Commented code, user or delete!

File: app/src/main/res/layout/item_details.xml [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <ScrollView
+    <android.support.design.widget.AppBarLayout
         android:layout_width="match_parent"
-        android:layout_height="match_parent">
+        android:layout_height="192dp">
```
Comment (JamesScottTAB) [inline app/src/main/res/layout/item_details.xml:10]: Hardcoded height

File: app/src/main/res/layout/item_details.xml [hunk 3]
```diff
@@ -1,36 +1,36 @@
...
-                    android:minHeight="200dp" />
-
-                <!-- TODO do not scroll toolbar (we need to handle toolbar transparency change on scroll first) -->
-                <android.support.v7.widget.Toolbar
-                    android:id="@+id/toolbar"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:padding="16dp" />
-
+                    android:minHeight="200dp"
```
Comment (JamesScottTAB) [inline app/src/main/res/layout/item_details.xml:32]: ++

File: app/src/main/java/com/nabilhachicha/kc/ItemDetailActivity.java (hunk 1 above)
Comment (kleinsenberg) [inline app/src/main/java/com/nabilhachicha/kc/ItemDetailActivity.java:77]: Will be used in slice 2. Simplified for now

File: app/src/main/res/layout/item_details.xml (hunk 2 above)
Comment (kleinsenberg) [inline app/src/main/res/layout/item_details.xml:10]: Pretty much all the stuff is hardcoded for now, will change this later

**Pull request merged**